policyengine_canada/parameters/gov/provinces/nu/tax/benefits/nucb/base/amount.yaml

```
@@ -0,0 +1,11 @@
+description: Nunavut provides this base component's base for each child in one family as part of the chile benefit.
+values:
+  2022-07-01: 330 #per child under age of 18 living at home
```

Per yaml style guides like https://developers.home-assistant.io/docs/documenting/yaml-style-guide/#comments
```suggestion
  2022-07-01: 330 # Per child under age of 18 living at home
```